src/main/java/org/rev317/min/Loader.java

```diff
@@ -36,6 +36,7 @@ public static Client getClient() {
     @Override
     public Applet fetchApplet() {
         try {
+            Core.verbose("[317-api-minified provider] Fetching Applet from class: "+Context.getInstance().getServerProviderInfo().getClientClass());
```

Comment:
This change would also need to be added to all other servers. Do we really need this?